jenkins/Pipeline.gy

```diff
@@ -50,8 +50,7 @@ pipeline {
             }
             steps {
                 sh 'buildah login -u="${PB_CI_REG_CRED_USR}" -p="${PB_CI_REG_CRED_PSW}" ${PB_CI_REGISTRY}'
-                sh 'bash -ex ./server/pbenchinacan/container-build-ci.sh'
-                sh 'buildah push localhost/pbench-server:${PB_IMAGE_TAG} ${PB_CI_REGISTRY}/${PB_ORG_NAME}/pbench-server:${PB_IMAGE_TAG}'
```

Comment:
Why did you remove the pbench-server image push to the ci registry? container-build.sh commits the image but don't we want to push it if we want to use it locally?

Reply:
There's _another_ `push` on the next line...and, since we have only one image now, we don't need to push it twice.  😁

Reply:
Ah, GitHub didn't show me the next line.